build_windows.cmd

```diff
@@ -10,13 +10,14 @@ mkdir build
 cd build
 
 set ARCHITECTURE=x64
-rem set BUILD_TYPE=Debug
+rem set BUILD_TYPE=Release
 set BUILD_TYPE=Release
-set VCPKG_PATH=C:\src\vcpkg
+set VCPKG_PATH=C:\DarknetOpenCV\vcpkg
 rem set TRIPLET=x64-windows-static
 set TRIPLET=x64-windows
+set CUDA_BIN_PATH="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8/bin"
```

Comment:
Oh, that's unfortunate.  CUDA and CUDNN are such a pain in the rear.  Hard-coding the version number into the path means everyone who needs to use this will have to modify the path.